java/arcs/core/analysis/PolicyVerifier.kt

```diff
@@ -40,7 +40,7 @@ class PolicyVerifier(val options: PolicyOptions) {
 
         // Compute the egress check map.
         val egressCheckPredicate = policyConstraints.egressCheck
-        val egressParticles = recipe.particles.filterNot { it.spec.isolated }
+        val egressParticles = recipe.particles.filterNot { it.spec.dataflowType.isolated }
```

Comment:
Shouldn't this be `filter { ...dataflowType.egress }`?

Reply:
Good catch, thank you!